rust/libnewsboat/src/utils.rs

```diff
@@ -596,6 +596,14 @@ mod tests {
         assert!(!is_exec_url("   exec:"));
     }
 
+    #[test]
+    fn t_get_default_browser() {
+        std::env::set_var("BROWSER", "firefox");
```

Comment:
This modifies the variable for the whole process, and might affect other tests. That's why we test this functions in a separate process, see rust/libnewsboat/tests/get_default_browser.rs. Please remove this commit.

Reply:
Oh, I did not noticed that the tests are separated for that function, thanks for pointing out.